org.eclipse.lsp4j/src/test/xtend-gen/.gitignore

```diff
@@ -1,2 +0,0 @@
-*
-!.gitignore
```

Comment:
Please don't remove this .gitignore file, it's used to prevent checking the generated Java test code into Git. The .java files in `src/test/xtend-gen` should not be committed.

Reply:
My bad, I'll revert that

Reply:
Done